katalog/fluentd/deploy.yml

```diff
@@ -1,10 +1,11 @@
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
```

Comment:
Any reason for adopting a Deployment instead of a Statefulset?
I've seen fluentd in this configuration usually deployed as the latter to allow persistent buffering from fluentd.

Reply:
As soon as i understand how buffering works, i will change it as statefulset

Reply:
in the eleasticsearch output configuration there is the buffer section (https://docs.fluentd.org/configuration/buffer-section), as you can see currently we are using in memory buffer, but we could use files and move to statefulset to be sure no logs are lost after restarts.

Reply:
done!